sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/PartitioningUtils.scala

```diff
@@ -144,7 +149,7 @@ private[sql] object PartitioningUtils {
    *       Literal.create("hello", StringType),
    *       Literal.create(3.14, FloatType)))
    * }}}
-   * and the base path:
+   * and the path when we stop the discovery is:
    * {{{
    *   /path/to/partition
```

Comment:
Should we add the `hdfs://<host>:<port>` part? I think `basePath` is required to be a canonical/qualified HDFS path, would be better to document this explicitly.